### DISCUSSION TITLE
chapter_greedy/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#635] [Announcements]
# chapter_greedy/

动画图解、一键运行的数据结构与算法教程

https://www.hello-algo.com/chapter_greedy/

**Comment by Osub** [comment]
配图使用AI生成的吗

**Reply by krahets** [reply]
yep

**Comment by realwaterking** [comment]
有kmp算法吗

**Reply by krahets** [reply]
暂时没有

**Reply by JiajunWu-XJTU** [reply]
要是能加上就好啦，感觉在串里面还是挺重要的，不过还是很感谢大大！

**Comment by pyyo** [comment]
不错哦

**Comment by whh13345** [comment]
看着Abstract莫名感觉像规则怪谈hhh

**Comment by JuiceJZZ** [comment]
终于看到尾声了，对于算法初学者来说，其实还有很多部分不曾理解，打算先刷刷题目沉淀一下，等到学期大半过去后再次重看它！！！

**Reply by zengxmy** [reply]
错了，先学习方法，再刷题，否则刷100遍也只是数量+1

**Comment by jingshao-code** [comment]
看完啦！谢谢笔者！写的非常好！
